apps/studio/components/interfaces/Auth/Hooks/AddHookDropdown.tsx

```diff
@@ -80,43 +85,37 @@ export const AddHookDropdown = ({
       </DropdownMenuTrigger>
       <DropdownMenuContent className="w-76" align={align}>
         <div>
-          {nonEnterpriseHookOptions.map((h) => (
+          {availableHooks.length === 0 && (
+            <DropdownMenuLabel className="text-foreground-light">
+              All available hooks have been added
+            </DropdownMenuLabel>
+          )}
+          {availableHooks.map((h) => (
             <DropdownMenuItem key={h.title} onClick={() => onSelectHook(h.title)}>
               {h.title}
             </DropdownMenuItem>
           ))}
         </div>
+        {nonAvailableHooks.length > 0 && (
+          <>
+            <DropdownMenuSeparator />
```

Comment:
I think the separator would still show if `availableHooks` is empty, perhaps we show only add the separator if both lists are non-empty?